src/main/java/frc/robot/Robot.java

```diff
@@ -45,7 +50,7 @@ public void robotInit() {
 
     @Override
     public void configureButtonBindings() {
-
+        controller.a().onTrue(arm.moveToDistance(200, 0.3));
```

Comment:
Lets not map things to buttons yet. Instead we should send this command to the smartdashboard.